frontend/components/shelter/pet/PetDetailsSection.tsx

```diff
@@ -1,53 +1,65 @@
 import { EditOutlined } from "@ant-design/icons";
```

Comment:
updates to this file are to facilitate removal of PetData from enum/types and to use Animal.Attributes from contract instead - in preparation of getting data from API